### ISSUE TITLE
Package 'fuse' and package 'sshfs' -- what are the differences?

### ISSUE THREAD

**Issue by POTSandSTOP** [libfuse/sshfs#175]
I'm attempting to repurpose an old Raspberry Pi. After doing `sudo apt-get update`, I tried:

~~~~
raspberrypi:~ pi$ sudo apt-get install sshfs
     Reading package lists... Done
     Building dependency tree
     Reading state information... Done
     E: Unable to locate package sshfs
raspberrypi:~ pi$ apt-cache search sshfs
     (no results, not even 'sshfs-fuse' which I've seen referenced elsewhere on the web)
raspberrypi:~ pi$ apt-cache search fuse
     libfuse2 - Filesystem in Userspace (library)
     psmisc - utilities that use the proc file system
     gvfs-backends - userspace virtual filesystem - backends
     gvfs-common - userspace virtual filesystem - common data files
     fuse - Filesystem in Userspace
     gvfs-libs - userspace virtual filesystem - private libraries
     gvfs - userspace virtual filesystem - GIO module
     gvfs-daemons - userspace virtual filesystem - servers
     gvfs-fuse - userspace virtual filesystem - fuse server
raspberrypi:~ pi$ sudo apt-get install fuse
     Reading package lists... Done
     Building dependency tree
     Reading state information... Done
     fuse is already the newest version.
     fuse set to manually installed.
     0 upgraded, 0 newly installed, 0 to remove and 26 not upgraded.
raspberrypi:~ pi$
~~~~

To put it another way, what is the relationship between 'fuse' and 'sshfs'? Do I give up anything if I use 'fuse' vs. 'sshfs'?

**Comment by Nikratio** [comment]
sshfs is using fuse. You can't use fuse on its own.
